packages/css/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@decathlon/vitamin-css",
+  "version": "0.0.1",
+  "description": "CSS styles for Vitamin Web",
+  "author": "Decathlon <designsystem@decathlo>",
```

Comment:
```suggestion
  "author": "Decathlon <designsystem@decathlon.com>",
```

Reply:
Thanks, Terence!
I cannot accept your commit suggestion because it's `@decathlon.net`.
I've just made the fix, thanks a lot!